emails/signals.py

```diff
@@ -27,5 +33,32 @@ def check_premium_for_block_list_emails(sender, instance, **kwargs):
                 raise SuspiciousOperation("Must be premium to set block_list_emails")
 
 
+@receiver(pre_save, sender=Profile)
+def measure_feature_usage(sender, instance, **kwargs):
+    if instance._state.adding:
+        # if newly created Profile ignore the signal
+        return
+    curr_profile = Profile.objects.get(id=instance.id)
+
+    # measure tracker removal usage
+    changed_tracker_removal_setting = (
+        instance.remove_level_one_email_trackers
+        != curr_profile.remove_level_one_email_trackers
+    )
+    if changed_tracker_removal_setting:
+        print(changed_tracker_removal_setting)
+        if instance.remove_level_one_email_trackers:
+            incr_if_enabled("tracker_removal_enabled")
+        if not instance.remove_level_one_email_trackers:
+            incr_if_enabled("tracker_removal_disabled")
+        info_logger.info(
+            "tracker_removal_feature",
+            extra={
+                "enabled": instance.remove_level_one_email_trackers,
+                "hashed_uid": sha256(instance.fxa.uid.encode("utf-8")).hexdigest(),
```

Comment:
Nit / Future PR: a utility function to create this hash, with tests

```python
class Profile:
   ...
   def hashed_uid(self):
       return sha256(self.fxa.uid.encode("utf-8")).hexdigest()
```

Reply:
Moving this as a TODO. Do you want me to create an issue?

Reply:
383fc57